types/react-redux/index.d.ts

```diff
@@ -577,7 +591,7 @@ export interface TypedUseSelectorHook<TState> {
  *   return <div>{store.getState()}</div>
  * }
  */
-export function useStore<S = any, A extends Action = AnyAction>(): Store<S, A>;
+export function useStore<S = SafeRootState, A extends Action = AnyAction>(): Store<S, A>;
```

Comment:
This is a breaking change now. I think you should keep all other places as `{}` while keeping this `any`

Reply:
Yeah this is a breaking change but I'm not sure why it's `any` in the first place. I feel like it should have been an empty object.

Reply:
That may be, but unfortunately there are already too many apps out there that depend on this. Therefore, we need to revert this change.